SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -683,3 +669,58 @@ void UGlobalStateManager::SetDeploymentMapURL(const FString& MapURL)
 	UE_LOG(LogGlobalStateManager, Log, TEXT("Setting DeploymentMapURL: %s"), *MapURL);
 	DeploymentMapURL = MapURL;
 }
+
+bool UGlobalStateManager::ProcessOpListForServersCanBeginPlay(const TArray<Worker_OpList*>& OpLists)
+{
+	const Worker_AddComponentOp* AddOp = nullptr;
+	const Worker_AuthorityChangeOp* AuthOp = nullptr;
+
+	for (Worker_OpList* OpList : OpLists)
+	{
+		for (size_t i = 0; i < OpList->op_count; ++i)
+		{
+			const Worker_Op* Op = &OpList->ops[i];
+
```

Comment:
In the case that bCanBeginPlay is false, we also need to know the upcoming authority state of the StartupActorManagerComponent (we could either be first startup on the authoritative server, or first start on the non-auth server that has received the component update before the first server has set CanBeginPlay = true)